server/api/controllers/posts.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const router = express.Router();
+
+const Post = require('../models/posts')
+
```

Comment:
I like that comments are added to specify what each route does, that makes life a lot easier. Also good to see use of Routers which is in line with the MVC pattern.